package.json

```diff
@@ -29,6 +29,7 @@
     "stackjs": "^0.1.0",
     "validator": "*",
     "webshot": "*",
-    "write-json-file": "^2.3.0"
+    "write-json-file": "^2.3.0",
+    "primordial": "*"
```

Comment:
It is a good practice to add dependencies in the alphabetical order to avoid duplicates when the list grows bigger. Please move `primordial` entry up and also fix the order of `rimraf` and `puppeteer`. Don't forget to remove trailing comma from the last item.